reference/widgets.v3.yml

```diff
@@ -2,7 +2,48 @@ swagger: '2.0'
 info:
   title: Widgets
   version: ''
-  description: 'Create and manage [widgets](/api-docs/storefront/widgets/widgets-overview), widget templates, regions, and placements.'
+  description: |-
+    Create and manage widget templates, widgets, regions, and placements.
+
+    - [Authentication](#authentication)
+    - [Subresources](#subresources)
+    - [Additional Information](#additional-information)
+
+    ## Authentication
+
+    Authenticate requests by including an [OAuth](https://developer.bigcommerce.com/api-docs/getting-started/authentication) `access_token` in the request header.
```

Comment:
Authenticate requests by including an [OAuth](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#obtaining-store-api-credentials) `access_token` in the request header.